manual/source/analyses/epf_dmg.md

```diff
@@ -17,9 +17,11 @@ key name | type | name | description
 --- | --- | --- | ---
 `result_name` <sup>*</sup> | `str` | Result name | Name of the result dataset.
 `hazard_type` <sup>*</sup> | `str` | Hazard type | Hazard type (earthquake, tsunami, tornado, hurricaneWindfields). 
-`hazard_id` <sup>*</sup> | `str` | Hazard id | ID of the hazard from the Hazard service.
+`hazard_id` <sup>*</sup> | `str` | Hazard id | Hazard ID which defines the particular hazard (e.g. New Madrid <br>earthquake using Atkinson Boore 1995).
 `fragility_key` | `str` | Fragility key | Fragility key used in mapping dataset.
-`use_liquefaction` | `bool` | Liquefaction | Use liquefaction, if applicable to the hazard. <br>Default is *False*.
+`liquefaction_fragility_key` | `str` | Liquefaction key | Use liquefaction, if applicable to the hazard. <br>Fragility key to use in liquefaction mapping dataset.
```

Comment:
Should this just say "Fragility key to use in mapping dataset for liquefaction"?

Reply:
I will change it.